README.md

```diff
@@ -1,38 +1,81 @@
-# Newdle
+## WARNING: newdle is not yet production ready
+### Currently missing
+ * [ ] E-mail sending;
+ * [ ] Integration with calendaring systems;
+ * [ ] Integration with Indico;
 
-This is something new and cool! 🎉
+If you would like to help with any of that, please contact us through the relevant GitHub issue.
 
-## Database
+# Newdle [![Travis Build Status](https://travis-ci.com/indico/newdle.svg?branch=master)](https://travis-ci.org/indico/newdle) [![License](https://img.shields.io/github/license/indico/newdle.svg)](https://github.com/indico/newdle/blob/master/LICENSE) [![Made at CERN!](https://img.shields.io/badge/CERN-Open%20Source-%232980b9.svg)](https://home.cern)
 
-To create the tables, run `flask db upgrade`.
+<img src="./newdle/client/src/images/logo.svg" width="300"><br>
+
+
+**Newdle** is the new, shiny tool brought to you by the [Indico Team](https://getindico.io/about) @ [CERN](https://home.cern) 🎉
+Here at CERN we **hold a lot of meetings**. While [Indico](https://getindico.io) makes it super easy to manage those meetings, we still **lose a lot of time** 📈 trying to schedule them, which usually involves numerous emails and private messages. That is what newdle has been created for: to streamline the process of choosing **the perfect date and time** 🗓 for your next meeting/event.
+
+Newdle is part of the [MALT project](https://malt.web.cern.ch).
+
+## Why another tool?
+
+
+It is true that there is an abundance of similar tools like e.g. [Nuages](https://nuages.domainepublic.net) or [croodle](https://github.com/jelhan/croodle) which are just two of many others available out there. Despite that, we still believe that newdle will provide users with more features and will offer much better user experience. We are putting huge efforts in order to be sure that newdle will meet the following requirements:
```

Comment:
It's quite nice, but I would give more importance to the items below. It looks almost as if our differentiating factor is UX and then those things are just extras...
Plus, I don't think there's an "abundance" of similar tools. I would say something like "there are several commercial projects (including Doodle, which was our main inspiration) and a handful of Open Source projects with similar functionality and in different states of development (or abandonment). However none of them provide bla bla bla..."